distribution/registry.go

```diff
@@ -123,9 +124,24 @@ func newRepository(
 			Class:      repoInfo.Class,
 		}
 
+		var newAuthTransport = authTransport
+		//TODO: Support multiple challenges for loading TLS configs
```

Comment:
```suggestion
		// TODO: Support multiple challenges for loading TLS configs
```
It's idiomatic to start human-readable comments with a space (in contrast to directive comments like `//nolint:staticcheck`).